3.0-HISTORY.rst

```diff
@@ -58,5 +58,8 @@
 
 - URLs are now automatically stripped of leading/trailing whitespace.
 
+- ``Response.raise_for_status()`` now returns the response object if the
+  response status is not erroneous.
```

Comment:
Heh, I know this is really nitpicking now, but can we rephrase this? It should be enough to say "now returns the response object if no exception is raised."